hadoop-hdds/docs/content/recipe/SparkOzoneFSK8S.md

```diff
@@ -64,18 +64,22 @@ Create a new directory for customizing the created docker image.
 
 Copy the `ozone-site.xml` from the cluster:
 
-```
+```bash
 kubectl cp om-0:/opt/hadoop/etc/hadoop/ozone-site.xml .
 ```
 
-And create a custom `core-site.xml`:
+And create a custom `core-site.xml`.
 
-```
+```xml
 <configuration>
     <property>
         <name>fs.o3fs.impl</name>
         <value>org.apache.hadoop.fs.ozone.BasicOzoneFileSystem</value>
     </property>
+    <property>
+        <name>fs.AbstractFileSystem.o3fs.impl</name>
+        <value>org.apache.hadoop.fs.ozone.OzFs</value>
```

Comment:
This should be BasicOzFs to match with BasicOzoneFileSystem.

Reply:
Thanks for the catch.